packages/union-component/src/components/DonationPhoneInput.tsx

```diff
@@ -21,7 +23,7 @@ const DonationPhoneInput: React.FC<InputHTMLAttributes<HTMLInputElement>> = ({
   const [value, setValue] = useState<string>(`${defaultValue}`);
 
   const handleOnChange = (phone: string) => {
-    setValue(phone);
+    if (NUMBER_REGEX.test(phone)) setValue(phone);
```

Comment:
I think you can rather use `pattern` from `input`. https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/pattern

Reply:
@duranmla  The `react-phone-input-2` API doesn't allow to pass a `pattern` attribute

Reply:
I found out there's no need to add this, the input already takes care of preventing the user typing letters or odd characters